css/custom.css

```diff
@@ -59,6 +59,18 @@ i.fa.fa-times:hover {
   color: red;
 }
 
+.languagepicker.roundborders.large .select-your-language:hover {
+  background-color: #80BE62 ;
+}
+
+.languagepicker.roundborders.large .hover-black:hover {
```

Comment:
I think, you can replace ".languagepicker.roundborders.large" with ".languagepicker"

Reply:
Yes I agree with you that this can be done. But there isn't any problem if we use what i have written.